app/main/views/briefs.py

```diff
@@ -0,0 +1,162 @@
+from flask import jsonify, abort, current_app, request
+
+from dmapiclient.audit import AuditTypes
+from .. import main
+from ... import db
+from ...models import User, Brief, AuditEvent
+from ...utils import (
+    get_json_from_request, json_has_required_keys, pagination_links,
+    get_valid_page_or_1, get_request_page_questions
+)
+from ...service_utils import validate_and_return_lot, validate_and_return_updater_request
+from ...brief_utils import validate_brief_data
+
+
+@main.route('/briefs', methods=['POST'])
+def create_brief():
+    updater_json = validate_and_return_updater_request()
+    page_questions = get_request_page_questions()
+
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ['briefs'])
+    brief_json = json_payload['briefs']
+
+    json_has_required_keys(brief_json, ['frameworkSlug', 'lot', 'userId'])
+
+    framework, lot = validate_and_return_lot(brief_json)
+
+    if framework.status != 'live':
```

Comment:
Ah OK, fair enough then. From the diff it looked like it was a completely new method.